runtime/src/main/java/com/tns/Runtime.java

```diff
@@ -155,17 +158,43 @@ public static boolean isInitialized()
 		return (runtime != null) ? runtime.isInitializedImpl() : false;
 	}
 
+	/*
+		This method initializes the runtime and should always be called first and through the main thread
+		in order to set static configuration that all following workers can use
+	 */
+	public static Runtime initializeRuntimeWithConfiguration(Configuration config) {
```

Comment:
Can we rename these to Runtime.init() and Runtime.initWithConfiguration() ?

@Plamen5kov